SoftLayer/CLI/securitygroup/event_log.py

```diff
@@ -0,0 +1,32 @@
+"""Get event logs relating to security groups"""
+# :license: MIT, see LICENSE for more details.
+
+import json
+
+import click
+
+import SoftLayer
+from SoftLayer.CLI import environment
+from SoftLayer.CLI import formatting
+
+COLUMNS = ['event', 'label', 'date', 'metadata']
+
+
+@click.command()
+@click.argument('request_id')
```

Comment:
TODO: I'm not really sure using the request_id here is useful, as that doesn't seem to be a very accessible piece of information for a user to get. Maybe it would be better to get the logs by the security group id?